plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
But why write unit tests when there are working integration tests. Infact you can write a single test with expected fields for each message and they can be checked. Why copy the same damn boilerplate over and over, such a waste of time. You can see that those tests are copy pasted code as each has an un-necessary import  and each the same method mis-spelled, `test_hash_expected_type` should be `test_has_expected_type`